canonical-web-and-design/ubuntu.com
# WIP Licensing updates

## Done

- [ ] redirect people from the dead /licensing page to /legal/intellectual-property-policy
- [ ] redirect line 44:about/about-ubuntu/licensing/? to /legal/intellectual-property-policy
- [ ] redirect line 663:project/about-ubuntu/licensing/? to /legal/intellectual-property-policy
- [ ] redirect line 788:ubuntu/licensing/? to /legal/intellectual-property-policy
- [ ] update /legal/launchpad-terms-of-service per the copy doc
- [ ] update /community/mission per the [copy doc](https://docs.google.com/document/d/1KREEL0iRsg7G1Sol4J5v7EraNljDTiIYMem5_rQLtKE/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Visit all relevant pages and see that you are correctly redirected 
**Note: Final 2 tasks still in progress**


## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10497
